packages/components/upload/src/upload-content.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div
-    :class="[ns.b(), ns.m(listType), ns.is('drag', drag)]"
+    :class="[
+      ns.b(),
+      ns.m(listType),
+      ns.is('drag', drag),
+      ns.is('disabled', disabled),
+    ]"
     tabindex="0"
```

Comment:
Maybe it would be better to set `tabindex` to `-1` when the setting is disabled, so that you don't need to add focus related styles.

Reply:
After setting the tabindex to -1, I found that the mouse could also focus, and then I solved the problem by setting "pointer events" to "none" and deleting the excess CSS